jsapp/js/account/security/sso/ssoSection.module.scss

```diff
@@ -0,0 +1,41 @@
+@use 'scss/_colors';
+@use 'scss/sizes';
+@use 'scss/libs/_mdl';
+
+.root {
+  margin-top: sizes.$x20;
+  margin-bottom: sizes.$x60;
+  padding-top: sizes.$x14;
+  display: flex;
+  border-top: sizes.$x1 solid;
+  border-color: colors.$kobo-gray-92;
+}
+
+.titleSection {
+  flex: 2;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.title {
+  margin: 0;
+  color: colors.$kobo-gray-40;
+  font-weight: 600;
+}
+
+.bodySection {
+  flex: 4;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.optionsSection {
+  text-align: right;
+  // FIXME: Why does text-align do nothing?
+  display: ruby;
```

Comment:
Same as above: either not use `display: ruby` (because possibly not proper usage) or use `ruby-align`